nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/FileTransfer.java

```diff
@@ -141,6 +141,13 @@ default String getAbsolutePath(FlowFile flowFile, String remotePath) throws IOEx
         .defaultValue("false")
         .allowableValues("true", "false")
         .build();
+    public static final PropertyDescriptor FOLLOW_SYMLINK = new PropertyDescriptor.Builder()
```

Comment:
Could you change to:
````java
...
.name("ftp-follow-symlink")
.displayName("Follow symlink")
...
````

Reply:
as this property is common for both ftp and sftp, will it be good if I mention only ftp-follow-symlink ? as earlier properties like recursive, file regex and other properties names kept generic hence I had put only Follow symlink.

Reply:
Submitted changes with commit 7fede6760b4ecc5980125f19d89588827690909b